src/cascadia/TerminalSettingsModel/WslDistroGenerator.cpp

```diff
@@ -35,7 +35,7 @@ std::wstring_view WslDistroGenerator::GetNamespace()
 // - <none>
 // Return Value:
 // - a vector with all distros for all the installed WSL distros
-std::vector<Profile> WslDistroGenerator::GenerateProfiles()
+std::vector<Profile> _legacyGenerate()

```

Comment:
interesting -- introducing a full fallback mode that is just "do the old thing." clever.